bitnami/grafana/README.md

```diff
@@ -186,6 +186,7 @@ This solution allows to easily deploy multiple Grafana instances compared to the
 | `dashboardsProvider.configMapName` | Name of a ConfigMap containing a custom dashboard provider                        | `""`                 |
 | `dashboardsConfigMaps`             | Array with the names of a series of ConfigMaps containing dashboards files        | `[]`                 |
 | `datasources.secretName`           | Secret name containing custom datasource files                                    | `""`                 |
+| `notifiers.notifiersName`          | Notifiers name containing custom configMap files                                  | `""`                 |
```

Comment:
This is not aligned with the **values.yaml**. Please use the [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm) to update the README.md since it does it for you automatically based on the values metadata.

Reply:
Sorry for that mistake. I've corrected that.